src/scss/style.scss

```diff
@@ -4,6 +4,7 @@
 @import 'main';
 
 // * COMPONENTS
+@import 'components/buttonhalloffame';
```

Comment:
Czy to Tobie działa?

Reply:
Tu na dole masz DETAILS, jak to klikniesz to sprawdzisz dlaczego testy nie przechodzą

![Screenshot_2021-12-07-02-17-38-839_com android chrome](https://user-images.githubusercontent.com/41823689/144948223-626e2c34-ff94-454e-ba20-43678daadde8.jpg)

Reply:
tak, testy już poszły. był błąd w metodzie getScores, dlatego expect nie byl taki sam jak to be.
Później okazało się też, że toBe to nie jest właściwa metoda dla porównywania obiektów, poprawna jest toStrictEqual